docs/en/writing-running-appium/server-args.md

```diff
@@ -40,20 +40,20 @@ All flags are optional, but some are required in conjunction with certain others
 |`--nodeconfig`|null|Configuration JSON file to register appium with selenium grid|`--nodeconfig /abs/path/to/nodeconfig.json`|
 |`-ra`, `--robot-address`|0.0.0.0|IP Address of robot|`--robot-address 0.0.0.0`|
 |`-rp`, `--robot-port`|-1|port for robot|`--robot-port 4242`|
-|`--chromedriver-port`|9515|Port upon which ChromeDriver will run|`--chromedriver-port 9515`|
-|`--chromedriver-executable`|null|ChromeDriver executable full path||
+|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|9515|Port upon which ChromeDriver will run|`--driver-args '{"uiautomator2": {"chromedriver-port": 9515}}`|
+|`--driver-args '{"uiautomator2": {"chromedriver-executable": '/path/to/file'}}`|null|ChromeDriver executable full path||
 |`--show-config`|false|Show info about the appium server configuration and exit||
 |`--no-perms-check`|false|Bypass Appium's checks to ensure we can read/write necessary files||
 |`--strict-caps`|false|Cause sessions to fail if desired caps are sent in that Appium does not recognize as valid for the selected device||
 |`--isolate-sim-device`|false|Xcode 6 has a bug on some platforms where a certain simulator can only be launched without error if all other simulator devices are first deleted. This option causes Appium to delete all devices other than the one being used by Appium. Note that this is a permanent deletion, and you are responsible for using simctl or xcode to manage the categories of devices used with Appium.||
 |`--tmp`|null|Absolute path to directory Appium can use to manage temporary files, like built-in iOS apps it needs to move around. On *nix/Mac defaults to /tmp, on Windows defaults to C:\Windows\Temp||
 |`--trace-dir`|null|Absolute path to directory Appium use to save ios instruments traces, defaults to <tmp dir>/appium-instruments||
 |`--debug-log-spacing`|false|Add exaggerated spacing in logs to help with visual inspection||
-|`--suppress-adb-kill-server`|false|(Android-only) If set, prevents Appium from killing the adb server instance||
+|`--driver-args '{"uiautomator2": {"suppress-adb-kill-server": false}}'`|false|(Android-only) If set, prevents Appium from killing the adb server instance||
```

Comment:
`suppressAdbKillServer`

Reply:
it makes sense to either set default capabilities in args rather than set each value separately. It will also help with input validation